crates/bitwarden-wasm-internal/src/init.rs

```diff
@@ -12,26 +19,35 @@ pub enum LogLevel {
 
 fn convert_level(level: LogLevel) -> Level {
     match level {
-        LogLevel::Trace => Level::Trace,
-        LogLevel::Debug => Level::Debug,
-        LogLevel::Info => Level::Info,
-        LogLevel::Warn => Level::Warn,
-        LogLevel::Error => Level::Error,
+        LogLevel::Trace => Level::TRACE,
+        LogLevel::Debug => Level::DEBUG,
+        LogLevel::Info => Level::INFO,
+        LogLevel::Warn => Level::WARN,
+        LogLevel::Error => Level::ERROR,
     }
 }
 
-#[wasm_bindgen]
-pub fn set_log_level(level: LogLevel) {
-    let log_level = convert_level(level);
-    set_max_level(log_level.to_level_filter());
-}
-
 #[allow(missing_docs)]
 #[wasm_bindgen]
 pub fn init_sdk(log_level: Option<LogLevel>) {
     console_error_panic_hook::set_once();
+
     let log_level = convert_level(log_level.unwrap_or(LogLevel::Info));
-    if let Err(_e) = console_log::init_with_level(log_level) {
-        set_max_level(log_level.to_level_filter())
-    }
+
+    let filter = EnvFilter::builder()
+        .with_default_directive(log_level.into())
+        .from_env_lossy();
+
+    let fmt = tracing_subscriber::fmt::layer()
+        .with_ansi(false) // Only partially supported across browsers
+        .with_timer(UtcTime::rfc_3339())
```

Comment:
Please disable timer. It's crashing the web environments as `std::time` is not available. Locally testing it seems to work after disabling timer. (You have to explicitly disable it)

Reply:
resolved in https://github.com/bitwarden/sdk-internal/pull/550/commits/726db7c1f9fce027a7410eb5df7d89601332e032